docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+# DO NOT EDIT
+# The .env file has everything you need to edit.
+# Run options:
+# 1. Use prebuilt images (preferred method):
+#   run cmd: docker-compose up -d
+# 2. Build images on your own machine:
+#   build cmd: COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
+#   run cmd: docker-compose up -d
+
+version: '3.8'
+services:
+  proxy:
+    image: nginx
+    restart: always
+    volumes:
+      - ./proxy/nginx.conf:/etc/nginx/nginx.conf:ro
```

Comment:
We should pull this from the repository, because this docker-compose file will be also used by end users. It should only take the local conf if it is in dev mode.

Reply:
I don't get what you mean. This repo should be for both end users and dev. End users will use "production mode" developers "dev mode"

Reply:
At the moment the user only has to download the docker-compose.yml file and the librephotos.env file. I want to keep it that way. Most users already struggle with docker-compose. I don't want to add git as a requirement too.

Reply:
Ok, moved the compose volume mount only to dev, and added a dockerfile to copy default config. Once we generate the proxy pre-built image using this dockerfile we will have to remember to replace the image name also in production dockerfile